### PR TITLE
Add icons

### PR DESCRIPTION
**Pull request by sandatjepil**
## Click on the template that fits your PR
* [**Icon addition**](?expand=1&template=icon_addition.md)
* [General or miscellaneous change](?expand=1&template=general_change.md)
* *[Blank](?expand=1&body=+) (not recommended)*

### CODE REVIEW

File: svgs/nu_online.svg
Comment (x9136) [inline svgs/nu_online.svg:1]: The size is a bit larger than needed, and it's odd that the ellipse has a different length and width.

Screenshot 2024-01-25 at 06 19 03
It's worth making the icon closer to the original to make it look more unique.

**Review by x9136 — CHANGES_REQUESTED** [review]
Hey! Thanks for contributing. The icons have the incorrect sizes, but there are other comments.

Don't forget to remove all unnecessary files from the PR.

File: .github/workflows/build_own_apk.yml
Comment (Goooler) [inline .github/workflows/build_own_apk.yml:1]: Don't mess the whole file.

**Comment by sandatjepil** [comment]
Thank you for replying, I'm currently new at github and don't know what to do next.
I actually hardcoded the svgs since there are no tools available in android (I don't have any PC to draw).
and sorry about changes in GH workflow, I want to build my own apk and post it to my telegram group for testing.

**Comment by x9136** [comment]
> I'm currently new at github and don't know what to do next.

I don't know if it's possible in graphic editors on phones to control all the parameters you need to create icons. Apparently, it would take a lot of work to do everything right in such conditions.

As for builds, in theory you can make them in your fork.

**Pull request closed without merging**